spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1341,7 +1341,7 @@ public void testParallelPartialProgressWithMaxFailedCommitsLargerThanTotalFileGr
             // Since we can have at most one commit per file group and there are only 10 file
             // groups, actual number of commits is 10
             .option(RewriteDataFiles.PARTIAL_PROGRESS_MAX_COMMITS, "20")
-            .option(RewriteDataFiles.PARTIAL_PROGRESS_MAX_FAILED_COMMITS, "0");
+            .option(RewriteDataFiles.PARTIAL_PROGRESS_MAX_FAILED_COMMITS, "1");
```

Comment:
since this is a one-liner, maybe we should backport this to Spark 3.4 in the same PR. Also Spark 4 is probably affected too?